packages/Breadcrumbs/stories/examples/Variations.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { Story, Tagline, Gap } from "storybook/assets/styles/common.styles";
+import Heading from "@paprika/heading";
+import Breadcrumbs from "../../src";
+
+const URL = "https://www.wegalvanize.com/";
+
+const ExampleStory = () => {
+  return (
+    <Story
+      css={`
+        h2 {
+          margin-top: 32px;
+          color: #999;
+        }
+      `}
+    >
+      <Heading level={1} displayLevel={2} isLight>
+        Breadcrumbs
+      </Heading>
+      <Tagline>
+        <big>
+          <strong>Variations</strong>
+        </big>{" "}
+        – Browse use cases + recipes
+      </Tagline>
+
+      <Gap />
+
+      <Heading level={2} displayLevel={4} isLight hasDivider>
+        All kinds
+      </Heading>
+
+      <Heading level={3} displayLevel={5} isLight>
+        Short version
+      </Heading>
+
+      <Breadcrumbs>
+        <Breadcrumbs.Link href={URL}>Breadcrumb 1</Breadcrumbs.Link>
```

Comment:
I wonder if the BreadCrumb it should be a and Item that can be type link, can be the case when you want the Breadcrumb item to behave like a button instead of a link?

Reply:
I can make href optional then

Reply:
Addressed! resolve the conversation